mod/update_network.php

```diff
@@ -12,14 +12,15 @@
 
 function update_network_content(App $a)
 {
-	$profile_uid = intval($_GET["p"]);
+	$profile_uid = intval($_GET['p']);
+	$parent = intval($_GET['item']);
```

Comment:
What if the GET parameter isn't set?

Reply:
It is always set: https://github.com/friendica/friendica/pull/4514/files#diff-a3a4de556a72540dcdb30e5cacd17de3R422